### PR TITLE
Update Spring Boot to 1.5.14

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

**Comment by gastaldi** [comment]
[test]

File: spring-boot/current-community/cache/booster.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@ metadata:
   app:
     launcher:
       runsOn:
-        - '!starter'
+      - '!starter'
```
Comment (gastaldi) [inline spring-boot/current-community/cache/booster.yaml]: This seems weird, are you sure it's below `runsOn`?

Comment (metacosm) [reply spring-boot/current-community/cache/booster.yaml]: I didn't touch that, the YAML parser did so I assume it did what it thought was correct but let me check nevertheless…

Comment (metacosm) [reply spring-boot/current-community/cache/booster.yaml]: According to https://www.json2yaml.com/, it seems OK. (I hate YAML for this kind of crap)

Comment (gastaldi) [reply spring-boot/current-community/cache/booster.yaml]: It seems that it is generated below `launcher` but if you say it's ok, I'll take your word on it :)

**Comment by metacosm** [comment]
@gastaldi aren't the files validated anyway?

**Comment by gastaldi** [comment]
@metacosm only the structure, not the contents

**Comment by geoand** [comment]
@metacosm [This](https://github.com/fabric8-launcher/launcher-booster-catalog/pull/225) launcher PR if/when introduced will validate the contents of the catalog as well

**Comment by Ladicek** [comment]
To _some_ extent, that is :-)

**Comment by geoand** [comment]
@Ladicek I would say that it validates the most common error that (we at least) can encounter

**Pull request merged**